pkg/operator/configobservation/images/observe_images.go

```diff
@@ -148,6 +148,46 @@ func ObserveAllowedRegistriesForImport(genericListers configobserver.Listers, re
 	return observedConfig, errs
 }
 
+func ObserveImagestreamImportMode(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
+	listers := genericListers.(configobservation.Listers)
+	var errs []error
+	prevObservedConfig := map[string]interface{}{}
+
+	// first observe all the existing config values so that if we get any errors
+	// we can at least return those.
+	imageStreamImportModePath := []string{"imagePolicyConfig", "imageStreamImportMode"}
+	currentImageStreamImportMode, _, err := unstructured.NestedString(existingConfig, imageStreamImportModePath...)
+	if err != nil {
+		return prevObservedConfig, append(errs, err)
```

Comment:
nit: returning between `prevObservedConfig` and  `existingConfig` is a little confusing to the reader, we are basically saying here `return map[string]interface{}{}, append(errs, err)`

Reply:
done.